### DISCUSSION TITLE
Suggestion. Tic-Tac-Toe game

### DISCUSSION THREAD

**Discussion by kirillPro2020** [Monika-After-Story/MonikaModDev#7837] [general - submod / external content]
I can implement this game.
(But which Python?).

**Comment by Rai99** [comment]
I believe MAS uses python 2.

**Comment by kirillPro2020** [comment]
Ok

**Comment by kirillPro2020** [comment]
Let's say. I made it, but it's a simple game, not a mod
(I don’t know how to import and configure it for the mod).

**Comment by xXMidnight** [comment]
Techncially if you were very skilled at tic tac toe there would be a draw everytime. It's like pitting two tic tac toe computers together that are like super good and they keep drawing. Unless you put a 5x5 ( ͡° ͜ʖ ͡°)
